CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Unreleased
```

Comment:
Should we add changelogs from PRs that were merged recently?

Reply:
I was planning to do this separately in case there are overlaps and something lands before this one does.